apps/src/lib/levelbuilder/course-editor/CourseEditor.js

```diff
@@ -337,7 +340,10 @@ class CourseEditor extends Component {
             updatePublishedState={publishedState =>
               this.setState({publishedState})
             }
-            preventCourseVersionChange={this.props.preventCourseVersionChange}
+            preventCourseVersionChange={
```

Comment:
Are we concerned about the situation where someone sets just the family name and not the version year?

Reply:
I think we want people to set both at the same time right? Ill add some validation on that